### PR TITLE
add type definition

### PR DESCRIPTION
**Pull request by taoqf**

### CODE REVIEW

**Comment by Rycochet** [comment]
Should reference to #238 and #177 here, and maybe compare contents of my gist linked in it.

**Comment by taoqf** [comment]
I just want help, and I am using this project in my work.

**Comment by kachkaev** [comment]
Having typings for this library would be awesome! @guyonroche 🙏

File: package.json [hunk 1]
```diff
@@ -98,4 +99,4 @@
     "LICENSE",
     "README.md"
   ]
-}
+}
```
Comment (kachkaev) [inline package.json]: ouch 😆

Comment (taoqf) [reply package.json]: Sorry for the formatter

**Comment by guyonroche** [comment]
Thanks @taoqf for doing this!
I'm not the hugest fan of types in JS but I know a lot of people are :-) so this will be a great addition

**Comment by Purus** [comment]
How should I use this typings for my Typescript application?

**Comment by taoqf** [comment]
Just install this lib will be OK, after new version if published.

**Comment by taoqf** [comment]
ask @guyonroche

**Comment by ahaverty** [comment]
@taoqf @guyonroche It looks like this is in the repo and released 👏 

But on v1.0.1 `import * as ExcelJs from 'exceljs';` isn't giving me any types as expected.
I can't find a index.d.ts in the dist after installing either
Has anyone gotten it to work? 🤔

**Comment by taoqf** [comment]
@guyonroche , please put index.d.ts in release.

**Comment by grbspltt** [comment]
@ahaverty I found the typings in the repo, copied and pasted into the node_modules/exceljs folder.  Not a great solution but works until the release is fixed.

**Comment by spacem** [comment]
Before this change I was using types from https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/exceljs
ie. installed with `npm install @types/exceljs`

Now I updated to the latest exceljs and the 'official' types from this merge don't seem so good. To give one example the font interface says that font requires all the parameters where I believe actually they should be optional?

**Comment by kachkaev** [comment]
@spacem could you please submit a PR that would fix the typing issues you are facing?

**Comment by taoqf** [comment]
@spacem anything else?

**Pull request merged**